website/docs/components/accordion/partials/code/how-to-use.md

```diff
@@ -196,6 +196,42 @@ The `ariaLabel` value is applied to the HTML button which controls visibility of
   </Hds::Accordion>
 ```
 
+### titleTag
+
+The `titleTag` value changes the HTML element wrapping each of Accordion Item's toggle block. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if an Accordion is within a subsection of the page below a heading level 2, the value should be `"h3"`. 
+
+```handlebars
+  <div class="doc-accordion-flex-layout">
+    <Hds::Text::Display @tag="h2" @size="300">Examination period</Hds::Text::Display>
+  </div>
+  <Hds::Accordion @titleTag="h3" as |A|>
+    <A.Item>
+      <:toggle>Exam experience</:toggle>
+      <:content>
+        All certification exams are taken online with a live proctor, accommodating all locations and time zones. Online proctoring provides the same benefits of a physical test center while being more accessible to exam-takers. The live proctor verifies your identity, walks you through rules and procedures, and watches you take the exam. Learn more ways to prepare for an online proctored exam in our Knowledgebase.
+      </:content>
+    </A.Item>
+    <A.Item>
+      <:toggle>Requirements for attending an exam</:toggle>
+      <:content>
+        Before you register for an exam, review the Exam-taker Handbook to learn the requirements and policies for taking exams. It is your responsibility to know and abide by our program rules to successfully enter your exam appointment, failure to do so may result in forfeiture of appointment fees.        
+      </:content>
+    </A.Item>
+    <A.Item>
+      <:toggle>Your badge and certificate</:toggle>
+      <:content>
+        HashiCorp has partnered with Credly to offer you a digital badge and downloadable certificate upon passing a certification exam. There is no fee for this service and acceptance is up to you. Digital badges can be used in email signatures or digital resumes, and on social media sites such as LinkedIn, Facebook, and Twitter. Badges link back to a real-time verification feature that describes your qualifications.
+      </:content>
+    </A.Item>
+  </Hds::Accordion>
+```
+
+!!! Insight
+
+The default `titleTag` is `"div"` because the correct value is dependent on the individual page. We highly encourage consumers to update the `titleTag` because it makes what screen readers announce to users match the visual experience.
```

Comment:
```suggestion
The default `titleTag` is `"div"` because the correct value is dependent on the individual page. We strongly encourage consumers to update the `titleTag` to meet WCAG Success Criterion [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) as the visual experience should match what is presented to the user with assistive technology.
```